releasenotes/notes/swap-strategies-3ab013ca60f02b36.yaml

```
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    New transpiler passes have been added. The transpiler passes
+    ``SwapStrategyRouter`` is a base class for transpiler passes that use swap
+    strategies to route a block of commuting gates to the coupling map. Indeed,
+    routing is a hard problem but is significantly easier when the gates
+    commute. This is often found in variational algorithms such as QAOA. Such
+    cases can be dealt with using swap strategies that apply a predefined set
+    of layers of SWAP gates. The child class ``PauliEvolutionGateRouter`` is
```

You can reference to the classes with `:class:` prefix.
```suggestion
    of layers of SWAP gates. The child class :class:`qiskit.transpiler.passes.routing.swap_strategies.swap_strategy_router.PauliEvolutionGateRouter` is
```